### PR TITLE
Database schema and the connections files

### PR DESCRIPTION
**Pull request by MohammedYehia**
- Define general ENUM TYPE for:
  - [x] regions
  - [x] UK_cities
  - [x] trade_type
  - [x] turnover_size
  - [x] social_mission
- Define the following tables:
  - [x] Social Enterprise basic informations
  - [x] Social Enterprise details
  - [x] social impact list
  - [x] policy
  - [x] contract
- [x] The connections files

### CODE REVIEW

**Comment by sultanassi95** [comment]
Good PR description. as a start :clap:

File: server/database/db.sql [hunk 1]
```diff
@@ -0,0 +1,112 @@
+BEGIN;
+
+DROP TABLE IF EXISTS social_enterprise_basic, social_enterprise_details, contract, uk_cities, policy, social_impact_list;
```
Comment (sultanassi95) [inline server/database/db.sql]: Hi,

`uk_cities` is a TYPE right? shouldn't be removed from this statement?
if not, please explain.

Comment (sultanassi95) [reply server/database/db.sql]: Good job, your writing style is cool so far.
Can you keep it that way?

Comment (sultanassi95) [reply server/database/db.sql]: Look at the review comment, if you edit this assign me back to review @MohammedYehia

Comment (MohammedYehia) [reply server/database/db.sql]: yes I will add drop statement for the types

**Pull request merged**